torchvision/datasets/_optical_flow.py

```diff
@@ -139,7 +146,7 @@ def __init__(self, root, split="train", pass_name="clean", transforms=None):
                 if split == "train":
                     self._flow_list += sorted(glob(str(flow_root / scene / "*.flo")))
 
-    def __getitem__(self, index):
+    def __getitem__(self, index: int) -> Tuple:
```

Comment:
Could you make this more precise? Same for all other occurrences of plain `Tuple`